src/docfx/Program.cs

```diff
@@ -54,6 +54,15 @@ private static int ExecSubCommand(string[] args)
                 }
                 return 1;
             }
+            catch (InvalidOptionException e)
```

Comment:
>catch (InvalidOptionException e) [](start = 12, length = 32)

how about merge it with OptionParserException like catch (Exception ex) when (ex is OptionParserException|| ex is InvalidOptionException)